post-workshop/post-workshop-ja.md

```diff
@@ -1,52 +1,52 @@
 
 
-*Risks and Benefits*
-There are no direct risks or benefits to you from filling out this survey, and no compensation. We hope to use these results to improve workshops for future learners.
+*リスクとメリット*
+このアンケートにご協力いただくことによる直接的なリスクやメリットはなく、報酬もありません。この結果をもとに、今後の学習者のため、ワークショップの改善に役立てたいと考えています。
 
-*Consent*
-You are not required to take this survey to participate in our workshop. You may quit the survey at any time or skip any item other than those required to correctly sort your responses. 
+*承諾*
+このアンケートに回答することは、ワークショップに参加する条件ではありません。アンケートはいつ終了しても構いませんし、回答できる項目以外はスキップしていただいても構いません。
 
-If you have any questions about the study, please contact Erin Becker, Associate Director of The Carpentries at ebecker@carpentries.org or eribecker@ucdavis.edu or Megan Welsh, Assistant Professor of Education at the University of California, Davis at megwelsh@ucdavis.edu.
+この研究についてご質問がある場合は、Carpentriesのアソシエイト・ディレクターであるエリン・ベッカー（ebecker@carpentries.org または eribecker@ucdavis.edu）か、またはカリフォルニア大学デービス校の教育学助教授であるミーガン・ウェルシュ（megwelsh@ucdavis.edu）までご連絡ください。
 
-If you have any questions about your rights or treatment as a research participant in this study or would like to provide input about this research, please contact the University of California at Davis’ Institutional Review Board (IRB) at (916) 703-9151, IRBAdmin@ucdmc.ucdavis.edu, or 2921 Stockton Blvd, Suite 1400, Room 1429, Sacramento, CA 95817.
+本研究の参加者としての権利や待遇について質問がある場合、または本研究について意見を提供したい場合は、カリフォルニア大学デービス校のInstitutional Review Board（IRB）（電話：(916) 703-9151、IRBAdmin@ucdmc.ucdavis.edu、または2921 Stockton Blvd, Suite 1400, Room 1429, Sacramento, CA 95817）までご連絡ください。
 
 -----------
 
-### Do you consent to taking this survey? *
+### このアンケートを受けることに同意しますか？ *
 
-   - yes
-   - no
+   - はい
+   - いいえ
 
-### Are you 18 years of age or above? *
+### 18歳以上の方ですか？ *
 
-   - yes
-   - no
+   - はい
+   - いいえ
 
-### What is the identifier for your workshop? It is in the YYYY-MM-DD-sitename format and is used in the URL for the workshop website or the Etherpad you used.  Please copy and paste the workshop id into the text box below.  *
+### ワークショップのIDは何ですか？YYYY-MM-DD-sitename形式で、ワークショップのウェブサイトまたは使用したEtherpadのURLに使用されています。 以下のテキストボックスにワークショップIDをコピーして貼り付けてください。 *
 
 (short text answer)
 
-### Please enter a unique identifier as follows: Number of siblings (as numeric) + First two letters of the city you were born in (lowercase) + First three letters of your current street (lowercase).  
+### 固有の ID を以下のように入力してください。兄弟・姉妹の人数（半角数字）+ 生まれた街の最初の2文字（小文字）＋現在住んでいる通りの最初の3文字 (ローマ字小文字)。
 
-(short text answer)
+(短文回答)
 
-### The next set of questions is optional. If you prefer not to answer a question, please click the down arrow on your keyboard, or scroll down to navigate to the next question.   Please rate your level of agreement with the following statements. 
+### 次の質問は任意です。質問に答えたくない場合は、キーボードの下向き矢印をクリックするか、下にスクロールして次の質問に移動してください。  次の記述に対してどの程度同意するかを評価してください。
```

Comment:
This is a literal translation but @masamiy is right, keyboards don't "click". Migrates translation from #6.

```suggestion
### 以下の質問は任意です。 もし回答したくない場合はあなたのキーボードの下向き矢印押下、もしくは画面をスクロールして次の質問に進んでください。以下のステートメントの合意レベルを教えてください。
```